src/webfaf/templates/base.html

```diff
@@ -30,6 +30,12 @@
     {% endblock js %}
   </head>
   <body>
+      <!--Banner notifiying of scheduled downtime etc. Uncomment the below div and edit text as needed.-->
+      <!--
+      <div class="alert alert-danger">
+              <b>Attention all users, here be an important message.</b>
+      </div>
+      -->
```

Comment:
But... this will be shown to everyone. If you want to do that in upstream, can you wrap it in:

{%- if config['BANNER'] -%}

or you can steal the code from 

https://pagure.io/copr/copr/c/5ab2579adc40e39e056086babed5c1a608417666?branch=master